ftrr/src/main/java/framgia/co/edu/ftrr/controller/RequestController.java

```diff
@@ -0,0 +1,33 @@
+package framgia.co.edu.ftrr.controller;
+
+import framgia.co.edu.ftrr.dto.request.RequestDTO;
+import framgia.co.edu.ftrr.entity.Request;
+import framgia.co.edu.ftrr.service.RequestService;
+import framgia.co.edu.ftrr.util.ExcelUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.List;
+
+@RestController("/requests")
+public class RequestController {
+
+    @Autowired
+    private RequestService requestService;
+
+    @GetMapping
+    public List<RequestDTO> getAllRequest() {
+        return requestService.findAll();
```

Comment:
Bậy cái này nè
Em phải check quyền cái này á, tạo pull khác đi, pull này chỉ chứa import file request trainee thôi

Chỗ view request thì check quyền currentUser nhé

1. Coordinator:
+ Chỉ hiển thị request trong tháng hiện tại
+ Cho phép search theo tháng hoặc khoảng tháng  hoặc div

2. SM/Dm div
+ Hiển thị toán bộ request của Div mình quản lý
với độ ưu tiên là: trạng thái request: waitting, in progress, interview và done, pending(khi ko có trainee nào đi phỏng vấn hoặc Div ko có nhu cầu nữa), reject (Edu reject request của Div) sau đó là ngày tạo gần nhất

3. Hr
Chỉ hiển thị các request ở trạng interview và done thôi
Với độ ưu tiên là interview -> done; tiếp theo là ngày tạo gần nhất

4. SM Edu
Hiển thị toàn bộ request của các div
với độ ưu tiên là trạng thái request: waitting, in progress, interview và done, pending(khi ko có trainee nào đi phỏng vấn hoặc Div ko có nhu cầu nữa), reject (Edu reject request của Div) sau đó là ngày tạo gần nhất